packages/gatsby/gatsby-plugin-clipanion-cli/gatsby-node.js

```diff
@@ -54,6 +54,18 @@ exports.sourceNodes = ({actions, createNodeId, createContentDigest}, opts) => {
         `\`\`\`\n`,
       ].join(``));
 
+      if (command.options.length > 0) {
```

Comment:
Regarding the order, I think `details` and `extra options` should be next to each others. I remember the examples were put first because @Ayc0 felt it was the first thing people wanted to see. Perhaps we can try Examples / Details / Extra Options?

Reply:
See my question here regarding the "extra"-ness of these options: https://github.com/yarnpkg/berry/pull/1883#discussion_r495582021.

Because I don't see them as any more extra than the usage, I feel like shouldn't go after the details, as people wanting to know what the options of a command are probably wouldn't want to scroll down to the end of the page to see the options table.

Regarding them being after the examples, I don't have any strong preferences. My only concern is people having to scroll after the examples to see what a Command's options are, but commands generally don't have *that* many examples, so I'm fine with it.

Reply:
Before you need to know what the options are, you need to understand the core mechanisms. For instance, that's the order followed by most `man` pages ([rsync](https://linux.die.net/man/1/rsync), [wget](https://www.gnu.org/software/wget/manual/wget.html), [curl](https://curl.haxx.se/docs/manpage.html)).

Reply:
I can agree with that 👍. The problem remains that the details isn't only "the core mechanisms", but "the core mechanisms and how the various options affect the core mechanisms", like in `yarn add`'s case. This leads to situations where some options are described with many details in the `Details` section and then described in one sentence in the options table below. 🤔